packages/data/src/settings/actions.js

```diff
@@ -68,11 +69,17 @@ export function* persistSettingsForGroup( group ) {
 	}
 
 	// get data slice for keys
-	const dirtyData = yield select( STORE_NAME, 'getSettingsForGroupSlice', group, dirtyKeys );
+	const dirtyData = yield select( STORE_NAME, 'getSettingsForGroup', group, dirtyKeys );
 	const url = `${ NAMESPACE }/settings/${ group }/batch`;
-	const update = dirtyKeys.map( key => {
-		return { id: key, value: dirtyData[ key ] };
-	} );
+
+	// todo: update should be an array of objects like { id: key, value: dirtyData[ key ] }.
```

Comment:
In the blocks repo we follow the convention of doing `@todo` for todos' I'm guessing that might be different here but just mentioning in case.

For this todo, do you think there's enough info to help with future work on this (if it's done)?  For example, would any future developer be able to derive why this isn't being done now in this reducer?

Reply:
>do you think there's enough info to help with future work on this

Good question. No, it appears to me to be already addressed. It was leftover from the original copy/paste. I removed it.